web/styles/app_variables.css

```diff
@@ -655,13 +655,15 @@
     --banner-grid-template-rows-lg: 0.3125em auto 0.3125em; /* 5px at 16px/1em */
     --banner-grid-template-areas-md: ". . . . banner-close-button banner-close-button"
         ". . banner-label banner-label banner-close-button banner-close-button"
-        ". banner-action-buttons banner-action-buttons banner-action-buttons . .";
+        ". banner-action-buttons banner-action-buttons banner-action-buttons banner-close-button banner-close-button"
+        ". . . . banner-close-button banner-close-button";
```

Comment:
Is there a reason to have `banner-close-button` span two different columns? Assuming `var(--banner-horizontal-padding)` is what it says, it seems like the second instance could be placeheld `.` column, couldn't it?